modules/po/webadmin.bg_BG.po

```diff
@@ -192,13 +192,13 @@ msgid ""
 msgstr ""
 
 #: modules/po/../data/webadmin/tmpl/add_edit_network.tmpl:83
-msgid "Trust the PKI:"
+msgid "Automatically detect trusted certificates (Trust the PKI):"
```

Comment:
I don't think you should touch .po files at all in your PR. Automation will update po and pot automatically

Reply:
Got it. Will revise.